.travis.yml

```diff
@@ -45,6 +45,9 @@ cache:
 before_install:
   - export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-arm64"
   - export PATH="$JAVA_HOME/bin:$PATH"
+  - export MEMORYMB=2048
```

Comment:
I think 2048 is small value. The default one is 2500M.
Now Travis build hangs:
```
[INFO] Running org.apache.drill.exec.fn.impl.TestAggregateFunctions
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```
Possibly it is due to lack of heap memory.
@luocooong What do you think about reverting this value to the default one?

Reply:
#2548 
@luocooong pls check